### PR TITLE
Add Cello 2024 Q1/Annual report

### PR DESCRIPTION
**Pull request by yeasy**
Add Cello 2024 Q1/Annual report for TOC review

### CODE REVIEW

File: project-reports/2024/2024-annual-Hyperledger-Cello.md
Comment (petermetz) [inline project-reports/2024/2024-annual-Hyperledger-Cello.md:35]: @yeasy One thing that I've seen work well in the past is to open an issue in the GitHub issue tracker with the title "Who uses $PROJECT_NAME?" and then in there explain that you are collecting some information about users and their use-cases (as much or as little as they can/willing to volunteer).

Not a requirement nor a change request in the report, just an idea in case you were interested.

File: project-reports/2024/2024-annual-Hyperledger-Cello.md
Comment (petermetz) [inline project-reports/2024/2024-annual-Hyperledger-Cello.md]: @yeasy If there is a roadmap document or anything else that details what these releases are planned for (how many, what frequency, etc.) that I think could help a lot for new contributors/prospective users in evaluating the project. You might already have this document and if you do I would recommend linking to it right here in the annual report as well.
If you don't have such a document then maybe a few words here to expand on the details a little bit would help everyone get a more detailed picture about 2024.

File: project-reports/2024/2024-annual-Hyperledger-Cello.md
Comment (petermetz) [inline project-reports/2024/2024-annual-Hyperledger-Cello.md]: @yeasy Same thought as above in terms of expanding on some details here.

File: project-reports/2024/2024-annual-Hyperledger-Cello.md
Comment (petermetz) [inline project-reports/2024/2024-annual-Hyperledger-Cello.md:62]: @yeasy Have you considered organizing a workshop/meetup/event of any kind? I know from personal experience that the Linux Foundation staff is always **extremely** helpful with these sort of things. 

Just a thought, not a change request.

One more idea I have is [Hacktoberfest](https://hacktoberfest.com/) which tends to yield plenty of interest in the project that I am a maintainer of (Hyperledger Cacti)

**Review by petermetz — CHANGES_REQUESTED** [review]

**Review by denyeart — APPROVED** [review]

**Comment by arsulegai** [comment]
The team has showcased a commendable consistency in terms of bi-weekly meetings and participation. I have noticed that these meetings are well-recorded and have a regular attendance from a consistent group of participants. This, in conjunction with participation from at least three companies, is a positive sign of collaborative effort. 

Further, an increase in the issue count indicates improved documentation and increased activity. I appreciate the swift actions taken by the team, which reflects their dedication and commitment to the project.

However, there are areas that need improvement. There is a consistent delay in reporting to the TOC as seen in Q3 and Q4 of 2023. Despite the record of meetings showing active participation, the maintainers are not showing enough activity. There are also inconsistencies in quarterly reports, notably the missing V1.1.0 release in the annual report that was promised in the Q4 report. 

The activity in 2023 significantly reduced compared to 2022, which is a cause for concern. Additionally, the cello repository saw fewer code merges, with minor documentation related PRs being the only visible activity on GitHub. 

Certain strategic decisions also raise concerns, like the release of Fabric version 2.5 LTS while 3.0 is planned. Also, there is a noticeable skew with 60% of contributions coming from just two organizations and a similar trend is observed in contributors working outside of working hours.

To improve, I suggest aligning with the Fabric strategy and fostering collaboration with the Bevel project, which could bring in new contributors. I also recommend addressing the project's low legal compliance, as regular scan alerts indicate it as an area that needs immediate attention.

**Comment by ryjones** [comment]
@yeasy

File: project-reports/2024/2024-annual-Hyperledger-Cello.md
Comment (yeasy) [reply project-reports/2024/2024-annual-Hyperledger-Cello.md:35]: Good idea, we could track using the wiki page. Create one at https://github.com/hyperledger/cello/wiki/Who-Uses-the-Project.

File: project-reports/2024/2024-annual-Hyperledger-Cello.md
Comment (yeasy) [reply project-reports/2024/2024-annual-Hyperledger-Cello.md]: The roadmap has been proposed and discussed in the weekly meeting, and there's meeting note on the latest roadmap.
We are on the phase of migrating to latest fabric LTS of v2.5 in the year 2024.

File: project-reports/2024/2024-annual-Hyperledger-Cello.md
Comment (yeasy) [reply project-reports/2024/2024-annual-Hyperledger-Cello.md]: New features mean two part mainly:
* support chaincode operation
* support latest fabric LTS of v2.5

File: project-reports/2024/2024-annual-Hyperledger-Cello.md
Comment (yeasy) [reply project-reports/2024/2024-annual-Hyperledger-Cello.md:62]: Most issues are addressed in the weekly meeting now.
Recently there're new users to come to attend the meeting.
Open to the meetup idea too.

Comment (petermetz) [reply project-reports/2024/2024-annual-Hyperledger-Cello.md:62]: > Most issues are addressed in the weekly meeting now. Recently there're new users to come to attend the meeting. Open to the meetup idea too.

@yeasy KK, I'd recommend reaching out to David Boswell and Sean Bohan for the meetup/workshop organization.

File: project-reports/2024/2024-annual-Hyperledger-Cello.md
Comment (ryjones) [reply project-reports/2024/2024-annual-Hyperledger-Cello.md:35]: @yeasy please update the report with this

File: project-reports/2024/2024-annual-Hyperledger-Cello.md
Comment (ryjones) [reply project-reports/2024/2024-annual-Hyperledger-Cello.md]: @yeasy please update the report with this as well

File: project-reports/2024/2024-annual-Hyperledger-Cello.md
Comment (ryjones) [reply project-reports/2024/2024-annual-Hyperledger-Cello.md]: @yeasy please update the report with this as well

**Review by VRamakrishna — APPROVED** [review]
I'm adding my approval so the PR can be merged once @yeasy updates the report as per suggestions made by others.

**Review by petermetz — APPROVED** [review]

**Pull request merged**
